### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by Fathar010** [aviatbet/Aviator-predictor-app#2]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Prince-classic** [comment]
I can't download. Help with link

**Comment by musikaalvin** [comment]
It sucks like shit and nothing works. Can you be truthful for at least once

**Comment by Prince-classic** [comment]
Yes am truthful I have try but it's doesn't work help with the direct link to download please I beg

**Comment by Bishnubehera2464** [comment]
Aviator signal before 1wine app aviator game

**Comment by Prince-classic** [comment]
Anyone has the link please help
